internal/skia/skia_windows.go

```diff
@@ -1954,6 +1954,20 @@ func TextBlobBuilderAllocRun(builder TextBlobBuilder, font Font, glyphs []uint16
 	copy(((*[1 << 30]uint16)(unsafe.Pointer(buffer.Glyphs)))[:len(glyphs)], glyphs)
 }
 
+func TextBlobBuilderAllocRunPosH(builder TextBlobBuilder, font Font, glyphs []uint16, positions []float32, y float32) {
+	type textBlobBuilderRunBuffer struct {
```

Comment:
Rather than repeat this type, move it out and make it top-level so it can be shared.

Reply:
Makes sense, done.